data/functions/clean.sql

```diff
@@ -8,6 +8,16 @@ begin
 end
 $$ language plpgsql;
 
+
+-- There are weird explainations in the dimpreport table that appear in curly braces
+create or replace function clean_report(report text)
+returns text as $$
+begin
+	return regexp_replace(report, '\{[^)]*\}', '');
```

Comment:
What's the purpose of the `[^)]*` between curly braces? AFAIK, we can make do with a simpler pattern:

```
fec=> select rpt_tp_desc, regexp_replace(rpt_tp_desc, '{.*}', '') from dimreporttype where rpt_tp_desc like '%{%';
                             rpt_tp_desc                             |                 regexp_replace
---------------------------------------------------------------------+------------------------------------------------
 April Quarterly {One of 4 valid Report Codes on Form 5, RptCode}    | April Quarterly
 July Quarterly {One of 4 valid Report Codes on Form 5, RptCode}     | July Quarterly
 October Quarterly {One of 4 valid Report Codes on Form 5, RptCode}  | October Quarterly
 Year-end {One of 4 valid Report Codes on Form 5, RptCode}           | Year-end
 {Monthly} Year-end {Used by F3X and F3P monthly filers}             |
 24-Hour Report {One of 2 valid 24/48 Hour Codes on Form 5}          | 24-Hour Report
 48-Hour Report {One of 2 valid 24/48 Hour Codes on Form 5}          | 48-Hour Report
 July 15 (Q2) report {with coverage dates} and Semi-annual           | July 15 (Q2) report  and Semi-annual
 Semi-Annual only {no Q2 - no coverage dates}                        | Semi-Annual only
 Jan 31 Year End (YE) report {with coverage dates} and Semi-Annual   | Jan 31 Year End (YE) report  and Semi-Annual
 Semi-Annual only {No YE - no coverage dates}                        | Semi-Annual only
 July 31 Mid-Year (MY) report {with coverage dates} and Semi-annual  | July 31 Mid-Year (MY) report  and Semi-annual
 July 20 (M7) Monthly report {with coverage dates} and Semi-Annual   | July 20 (M7) Monthly report  and Semi-Annual
 Semi-Annual only {No M7 - no coverage dates}                        | Semi-Annual only
 Jan 31 Year End (MYE) report {with coverage dates} and Semi-annual  | Jan 31 Year End (MYE) report  and Semi-annual
  Semi-Annual only {No MYE - no coverage dates}                      |  Semi-Annual only
(16 rows)
```